app/src/PageController.php

```diff
@@ -12,37 +12,40 @@ class PageController extends ContentController implements TemplateGlobalProvider
     protected function init()
     {
         parent::init();
+        // these were from simple theme
+        // Requirements::themedCSS('reset');
+        // Requirements::themedCSS('layout');
+        // Requirements::themedCSS('form');
+        // Requirements::themedCSS('typography');
```

Comment:
Don't keep commented out code - if it's needed keep it (uncommented) and if it's not needed remove it.